### DISCUSSION TITLE
Can I stop a  suggestion on  `\\` (newline) ?

### DISCUSSION THREAD

**Discussion by kwangkim** [quarto-dev/quarto-cli#4357] [Q&A] [answered]
I am using quarto with many math formulas.
Whenever I type `\\`  for a new line, it keeps suggesting something.
Is there any way to stop a suggestion when I type `\\`?

![image](https://user-images.githubusercontent.com/10469752/218912586-e88cf5f3-277d-4902-a354-1fe73def9583.png)

**Comment by cderv** [comment, marked as answer]
See https://github.com/quarto-dev/quarto/issues/88 for tracking this. Thanks for the feedback
